### PR TITLE
change dropdown clear when doesn't have label

### PR DESCRIPTION
**Pull request by davidbgomes**
[Design: "Clear all" styles in resources should be the same as in policy explorer](https://app.clickup.com/t/865cg674b)

image

### CODE REVIEW

File: src/webapp/components/dropdown/Dropdown.tsx [hunk 1]
```diff
@@ -78,14 +78,15 @@ export const Dropdown: React.FC<DropdownProps> = React.memo(
     }
 );
 
-const ClearButton = styled(Button)`
+const ClearButton = styled(Button)<{ hasLabel: boolean }>`
```
Comment (xurxodev) [inline src/webapp/components/dropdown/Dropdown.tsx:81]: I think is clearest name this prop top

File: src/webapp/components/dropdown/Dropdown.tsx [hunk 2]
```diff
@@ -78,14 +78,15 @@ export const Dropdown: React.FC<DropdownProps> = React.memo(
...
     color: orange;
     text-transform: none;
     position: absolute;
     top: 0;
     right: 0;
     padding: 0px;
     font-weight: 400;
+    top: ${props => !props.hasLabel && "-30px"};
```
Comment (xurxodev) [inline src/webapp/components/dropdown/Dropdown.tsx:89]: Top css prop is duplicated and confuse. I think we unify setting parameter prop or 0

File: src/webapp/components/dropdown/Dropdown.tsx [hunk 3]
```diff
@@ -69,7 +69,7 @@ export const Dropdown: React.FC<DropdownProps> = React.memo(
                     onChange={(e, value) => handleChange(e, value as Option<string> | null)}
                 />
                 {value && (
-                    <ClearButton variant="text" onClick={handleClear}>
+                    <ClearButton variant="text" onClick={handleClear} hasLabel={!!label}>
```
Comment (xurxodev) [inline src/webapp/components/dropdown/Dropdown.tsx:72]: I think would be clearest if we set a top property and here set the logic to pass -30 if label is undefined

**Review by xurxodev — CHANGES_REQUESTED** [review]
thanks @davidbgomes 

I have minor comments to fix in the code

**Comment by davidbgomes** [comment]
Thank you for the review @xurxodev, I've made the changes to use the top prop instead. Should be good now.

**Review by xurxodev — APPROVED** [review]
thanks @davidbgomes

**Pull request merged**